### PR TITLE
Incorporating upstream changes to Kogito-docs 1.24.x

### PR DESCRIPTION
**Pull request by akumar074**
[Fix RHBQ procedure, add image procedure](https://github.com/kiegroup/kogito-docs/pull/145/commits/8ff0f10514b55576f271b7c31849346d07169190)

### CODE REVIEW

File: antora.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ asciidoc:
     quarkus_cli_platform_redhat: '-Pcom.redhat.quarkus.platform:quarkus-bom:2.7.6.Final-redhat-00006'
     kogito_version_redhat: 1.24.0.Final-redhat-00011
     quarkus_platform: com.redhat.quarkus.platform
-    kogito_sw_gav: >-
+    kogito_sw_ga: >-
```
Comment (ricardozanini) [inline antora.yml:14]: here we should remove the version.

Comment (akumar074) [reply antora.yml:14]: I have removed the version now. Thanks

**Comment by ljmotta** [comment]
@akumar074 Hi! could you please update the kn-cli-version to 0.21.3? The artifact isn't available yet but it will until EOD.

**Comment by akumar074** [comment]
Done @ljmotta

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**